.gitignore

```diff
@@ -43,3 +43,6 @@ jspm_packages
 .env*
 
 .cache
+
+# workspace config
+.vim
```

Comment:
I would prefer personal dev environment ignores to happen at a system level, outside the project.  See  https://jaketrent.com/post/gitignore-system-files/

Does that seem reasonable?

Reply:
good idea. i'll move it